uiowa/uiowa
# Wire up featured content heading size field.

Related to #2259 

This is just how my brain wanted to solve this. If there is a referring item going the other way around, that'd be fine too, just not seeing it.

### Testing
- `blt sync`
- Add a featured content block to page referencing >= 2 articles/people.
- Test out the usual headline field stuff:
  - With no headline value and different child heading sizes.
  - With a headline value and setting various sizes for it. Then inspect the children heading sizes.
  - Make sure the headline changes are as expected after saving the page.
